projects/aws/cluster-api-provider-cloudstack/Makefile

```diff
@@ -11,11 +11,18 @@ BINARY_TARGET_FILES=manager
 EXTRA_GO_LDFLAGS=$(shell $(BUILD_LIB)/version.sh $(REPO) aws/cluster-api-provider-cloudstack/pkg/version)
 
 IMAGE_COMPONENT=aws/cluster-api-provider-cloudstack/release/manager
+CAPC_MANIFESTS_TARGET=$(REPO)/pkg/mocks/mock_client.go
 
 HAS_S3_ARTIFACTS=true
 
 include $(BASE_DIRECTORY)/Common.mk
 
+$(REPO)/eks-anywhere-go-mod-download: $(CAPC_MANIFESTS_TARGET)
```

Comment:
In case anyone in future sees this, this is the first time we are having to generate something before mod download.  I dont love this, but well go with it for now until we decide we need a better pattern to follow, if it comes up more.